frontend/src/UserList.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react'
+
+import {
+  Badge,
+  Stack,
+  SimpleGrid,
+  Divider,
+  Box,
+  Text,
+  Button,
+  Icon,
+  InputGroup,
+  InputLeftElement,
+  Input,
+  PseudoBox,
+} from '@chakra-ui/core'
+
+import { Trans } from '@lingui/macro'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+import { PaginationButtons } from './PaginationButtons'
+
+export function UserList() {
+  // This function generates the URL when the page loads
+  const { loading, error, data } = useQuery(
+    gql`
```

Comment:
^^^
With a custom object this can be sooooooooooo much cleaner

Reply:
@IdezHD I think it would be better to update the faker file in a separate pull request and then issue an additional PR to fix all of the data bindings in the UI.